src/app/containers/ConsentBanner/Banner/index.amp.jsx

```diff
@@ -3,6 +3,7 @@ import { bool, string } from 'prop-types';
 import { ConsentBanner } from '@bbc/psammead-consent-banner';
 import { ServiceContext } from '#contexts/ServiceContext';
 import BannerText from './Text';
+import getDataAttribute from './getDataAttribute';
 
 const Accept = (message, onClick) => (
```

Comment:
It seems you skipped adding dataAttribute to the amp Banner.

Reply:
Added data attributes to amp Banner.